easybuild/easyconfigs/m/MUST/MUST-1.11.0-iimpi-2025b.eb

```
@@ -0,0 +1,53 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# Authors::   Robert Mijakovic <robert.mijakovic@lxp.lu>
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+# Authors::   Damian Alvarez <d.alvarez@fz-juelich.de>
+# Authors::   Benedikt Steinbusch <b.steinbusch@fz-juelich.de>
+# License::   New BSD
+#
+##
+easyblock = 'CMakeMake'
+
+name = 'MUST'
+version = '1.11.0'
+
+homepage = 'https://hpc.rwth-aachen.de/must'
+description = """MUST detects usage errors of the Message Passing Interface (MPI) and reports them
+to the user. As MPI calls are complex and usage errors common, this functionality is extremely helpful
+for application developers that want to develop correct MPI applications. This includes errors that
+already manifest --segmentation faults or incorrect results -- as well as many errors that are not
+visible to the application developer or do not manifest on a certain system or MPI implementation.
+"""
+
+toolchain = {'name': 'iimpi', 'version': '2025b'}
+
+source_urls = ['https://hpc.rwth-aachen.de/must/files/']
+sources = ['%(name)s-v%(version)s.tar.gz']
+patches = ['MUST-1.11.0_fix-mustrun.patch']
+checksums = [
+    {'MUST-v1.11.0.tar.gz': '9ebe0022b2bf6a6d39af52c8a363058777ce31838971123d5a51a193bcdfcae3'},
+    {'MUST-1.11.0_fix-mustrun.patch': 'aaf05b5b37c6ddcfb35cb1cb30b7d62dacf85f3f201b51d7898ffee685775e06'},
+]
+
+builddependencies = [
+    ('help2man', '1.49.3'),
+    ('git', '2.50.1'),
+]
+
+dependencies = [
+    ('CMake', '4.0.3'),
+    ('Graphviz', '13.1.2', '-minimal'),
+    ('libxml2', '2.14.3'),
+    ('Python', '3.13.5'),
+    ('elfutils', '0.193'),
+]
+
+preconfigopts = "rm -rf %(installdir)s && "
```

With https://github.com/easybuilders/easybuild-framework/pull/4932 in mind, would it be sufficient if we only *empty* an existing installation directory, rather than removing the directory itself?

```suggestion
preconfigopts = "rm -rf %(installdir)s/* && "
```

@Thyre What do you think?